docs/sparc.client.services.rst

```diff
@@ -4,6 +4,14 @@ sparc.client.services package
 Submodules
 ----------
 
+sparc.client.services.o2parc module
```

Comment:
"o2sparc" missing an "s" here.

Reply:
thanks for pointing this out. I have corrected it.